extension/src/popup/styles/global.scss

```diff
@@ -19,17 +19,13 @@
   --back--button-dimension: 1rem;
   --back--button-z-index: 2;
 
-  --background-color: var(--color-gray-00);
-  --background-color--overlay: var(--color-gray-10);
-  --secondary-text: var(--color-gray-70);
```

Comment:
interestingly - aliasing these colors doesn't work in scss. When you toggle between light/dark mode, it doesn't grab the respective value, it always just gets the light version. I couldn't quite figure out why this is, but it's probably okay because we wanted to move to a more declarative use of colors where components always just use the same color var names as listed in the figma rather than using a project-dependent alias